### DISCUSSION TITLE
Possible bug: "Send to" context menu in File Explorer on Windows 11 says "Electron" instead of "Positron".

### DISCUSSION THREAD

**Discussion by naltmann** [posit-dev/positron#5627] [Feedback]
The "Send to" menu has the expected Positron icon, but says "Electron". I assume that might be some type of default for apps made with Electron?

The menu is displayed when right-clicking on a file (e.g. an ".R" file) in File Explorer.

**Comment by jmcphers** [comment]
Thanks for reporting this! I can reproduce it, too. I've filed it as https://github.com/posit-dev/positron/issues/5633.
